server/game/player.js

```diff
@@ -40,7 +40,7 @@ class Player extends Spectator {
         this.selectedCards = [];
         this.selectableCards = [];
         this.cannotGainChallengeBonus = false;
-        this.promptedActionWindows = {
+        this.promptedActionWindows = user.promptedActionWindows || {
```

Comment:
Should we do an explicit `clone` here? As it currently stands, `this.promptedActionWindows` will reference the same object as `user.promptedActionWindows`. This means that as a player checks / unchecks things in a game, it will be modified on their user object as well, thus changing their account level preferences. Worse, even though it's changing their preferences, those changes will be lost the next the user object is loaded from scratch (guessing this only happens on initial log in). So we get this weird behaviour where 1. changes are being made to preferences without the user realizing it and 2. the changes are being reset at intervals not predictable to the user.